modules/acquia_cms_dam/config/pack_acquia_cms_dam/cohesion_templates.cohesion_content_templates.media_acquia_dam_image_asset_emb.yml

```diff
@@ -169,8 +169,7 @@ json_values: |
                       ]
                   },
                   "attributes": {
-                      "alt": "[media:image:alt]",
-                      "title": "[media:image:title]"
+                      "alt": "[media:acquia_dam_alt_text:value]"
```

Comment:
@deepakmishra2 - This will only be generated if there is a mapping for alt text field, otherwise it will be blank because default DAM mapping doesn't have alt text field mapping out of the box.
Hence, I would suggest to use field condition then display the fallback as file name as alt value.
cc @vishalkhode1